srfi-167.html

```diff
@@ -157,6 +157,11 @@ <h3 id="engine-unpack"><code>(engine-unpack engine bytevector)</code></h3>
     <p>Call an unpacking procedure that will decode a bytevector
         encoded with the above pack procedure into a Scheme object.</p>
 
+    <h3 id="make-default-engine"><code>(make-default-engine)</code></h3>
+
+    <p>Return an <code>engine</code> record for the current <code>okvs</code>
+        implementation.</p>
+
```

Comment:
`make-default-engine` is part of the "`okvs`" scheme library, not part of the `engine` library that is why it is documented in the okvs part. It provides the default engine implementation. Where engine is not specific to particular backend, the default engine is.

ref: https://github.com/scheme-requests-for-implementation/srfi-167/pull/9/commits/a6646be7f9a09c88788dfa86229752ace56ca933